### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by VSEphpbb**

### CODE REVIEW

File: styles/prosilver/template/ideas.js [hunk 1]
```diff
@@ -260,9 +260,11 @@
 			$.get(url, {duplicate: Number(value)}, function(res) {
 				if (res) {
 					if (value) {
-						$obj.duplicateLink.text('idea.php?id=' + value)
-							.attr('href', 'idea.php?id=' + value)
-							.show();
+						$obj.duplicateLink.text(function() {
+							return $(this).attr('data-l-msg');
+						})
+						.attr('href', '../app.php/idea/' + value)
```
Comment (michaelcullum) [inline styles/prosilver/template/ideas.js]: hardcoded app.php. This will break when rewriting is enabled.

Comment (VSEphpbb) [inline styles/prosilver/template/ideas.js]: no it won't

Comment (VSEphpbb) [inline styles/prosilver/template/ideas.js]: app.php is always there... the rewriting a syou know just affects if you try access the page without app.php in the url

Comment (michaelcullum) [inline styles/prosilver/template/ideas.js]: This breaks on .com, we've had issues in the past with it.

**Pull request merged**